src/main/resources/org/jenkinsci/plugins/envinject/EnvInjectJobProperty/help-propertiesContent.html

```diff
@@ -1,16 +0,0 @@
-<div>
-    <p>Give a set of key/value (one variable per line): <code>KEY=VALUE</code>.</p>
```

Comment:
Before there where duplicates in both the JobProperty and the BuildWrapper, since moving the config into the Info class some duplicates could be removed.